ghost/i18n/locales/es/search.json

```diff
@@ -1,9 +1,9 @@
 {
     "Authors": "Autores",
     "Cancel": "Cancelar",
-    "No matches found": "No se encontraron coincidencias",
+    "No matches found": "No se encontraron resultados",
     "Posts": "Publicaciones",
     "Search posts, tags and authors": "Buscar publicaciones, etiquetas y autores",
-    "Show more results": "Mostrar más resultados",
+    "Show more results": "Mostrar más",
```

Comment:
The button is the full width of the popup, so I think the original text would be fine here.

Reply:
Ok, then rollback to Mostrar más resultados.

Thanks for all your help in special.

Fernando Ochoa Olivares,
Thank you for supporting this project and rectifying the errors I made.

Reply:
Have been having a lot of workload, leaving the project, thanks for everything though, have three projects for the company I work with and will be programming almost all day, adding to that my job at NASA, and [https://github.com/pyca/cryptography](https://github.com/pyca/cryptography) my current workload is too much right now, perhaps I re-join when I have less work.

Thanks,
Fernando Ochoa Olivares.

MIT Researcher - https://orcid.org/0009-0000-8974-2462